openapi-cli/src/main/java/io/ballerina/openapi/cmd/OpenApiCmd.java

```diff
@@ -110,6 +110,10 @@ public class OpenApiCmd implements BLauncherCmd {
     @CommandLine.Option(names = {"--with-service-type"}, hidden = true, description = "Generate service type")
     private boolean generateServiceType;
 
+    @CommandLine.Option(names = {"--without-data-binding"}, hidden = true,
+            description = "Generate service without data binding")
+    private boolean generateWithoutDataBinding;
```

Comment:
Do we give a warning/error if the flag is used in the client generation mode?

Reply:
Added